apps/fabric-website-resources/webpack.serve.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = resources.createServeConfig({
   resolve: {
     alias: {
       'office-ui-fabric-react/src': path.resolve(__dirname, '../../packages/office-ui-fabric-react/src'),
-      'office-ui-fabric-react/lib': path.resolve(__dirname, '../../packages/office-ui-fabric-react/src'),
+      'office-ui-fabric-react/lib': path.resolve(__dirname, '../../packages/office-ui-fabric-react/lib'),
```

Comment:
This change prevents changes to examples from being reflected locally on live reload.